taskcluster/test/params/train.yml

```diff
@@ -0,0 +1,94 @@
+base_ref: origin/main
```

Comment:
Question: Since I'm new to the codebase, how are we generating or creating these train.yml files? Is this going to be a maintenance burden? I'm just unclear

Reply:
Suggestion: If these will get stale in the future we might include a comment at the top of how to "re-generate" them, or where to get a new one from.